cli/command_repository_create.go

```diff
@@ -2,8 +2,10 @@ package cli
 
 import (
 	"context"
+	"time"
 
 	"github.com/alecthomas/kingpin"
+	"github.com/minio/minio-go/v7"
```

Comment:
that is not a desireable dependency, can we hardcode the constant instead?

Reply:
Good point. We can make some constants yes. Will do.

Reply:
done